py4cast/datasets/__init__.py

```diff
@@ -33,13 +33,33 @@
 except (ImportError, FileNotFoundError, ModuleNotFoundError):
     warnings.warn(f"Could not import TitanDataset. {traceback.format_exc()}")
 
+try:
+    from .titan import InferTitanDataset
+
+    registry["titan_infer"] = (
+        InferTitanDataset,
+        default_config_root / "titan_infer.json",
+    )
+except ImportError:
+    warnings.warn(f"Could not import InferTitanDataset. {traceback.format_exc()}")
+
```

Comment:
I think you didn't want to commit this one ?

Reply:
No, indeed, it will be for the titan inference. I fixed it. Thank you.